ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListSourceExecutor.java

```diff
@@ -299,6 +304,10 @@ private static SourceDescriptionWithWarnings describeSource(
       );
       sourceConstraints = getSourceConstraints(name, ksqlExecutionContext.getMetaStore());
     } catch (final KafkaException | KafkaResponseGetFailedException e) {
+      if (Throwables.getRootCause(e) instanceof UnknownTopicOrPartitionException) {
+        LOG.warn("Failed to Describe due to UnknownTopicOrPartitionException for {} "
```

Comment:
```suggestion
        LOG.warn("Failed to Describe. [Error={}, Topic={}, Stream={}]", ...)
```

Reply:
Updated (Added Image in Description too) - This is how logger would look like:

2024-12-20 10:28:06 [2024-12-20 04:58:06,094] WARN Failed to Describe. [Error=org.apache.kafka.common.errors.UnknownTopicOrPartitionException: This server does not host this topic-partition., Topic=test_topic_2, Source=TEST_STREAM_2] (io.confluent.ksql.rest.server.execution.ListSourceExecutor:305)